easybuild/tools/repository/svnrepo.py

```diff
@@ -88,8 +88,8 @@ def setup_repo(self):
         try:
             pysvn.ClientError  # IGNORE:E0611 pysvn fails to recognize ClientError is available
         except NameError, err:
-            self.log.exception("pysvn not available (%s). Make sure it is installed " % err +
-                               "properly. Run 'python -c \"import pysvn\"' to test.")
+            self.log.exception("pysvn not available (%s). Make sure it is installed properly."
+                               + " Run 'python -c \"import pysvn\"' to test.", err)
```

Comment:
we don't use `log.exception` anywhere in the EasyBuild codebase, it's all `log.error`

`log.error` will result in a (somewhat) nice error message, which should certainly be sufficient in this case (the error message is quite clear)

`log.exception` would result in a traceback, which you don't want to expose users to

so, please change to `log.error` instead...

we can probably fix `log.error` to make it works like it should when not using `%` (but let's open a different PR for that)